statistics/histogram.go

```diff
@@ -296,6 +296,23 @@ func ValueToString(value *types.Datum, idxCols int) (string, error) {
 	return str, err
 }
 
+// IndexValueToString converts a possible encoded value to a formatted string. If the value is encoded, then
+// idxCols equals to number of origin values, else idxCols is 0.
+func IndexValueToString(value *types.Datum, idxCols int, idxColumnType []byte) (string, error) {
```

Comment:
How about adding this new argument `idxColumnType` to the original function `ValueToString` instead of creating a new one?
For example, `ValueToString(value *types.Datum, idxCols int, idxColumnType []byte)`, and the `idxColumnType` can be `nil`.